src/mono/mono/metadata/bundled-resources.c

```diff
@@ -62,48 +73,62 @@ bundled_resources_is_known_assembly_extension (const char *ext)
 #endif
 }
 
-static gboolean
-bundled_resources_resource_id_equal (const char *id_one, const char *id_two)
+// strrchr calls strlen, so we need to do a search with known length instead
+// for some reason memrchr is defined in a header but the build fails when we try to use it
```

Comment:
are you talking about emscripten specifically?  It looks like you need to define `_GNU_SOURCE` before including `<string.h>` to get it.  I thought our cmake already defined it, but looks like it's not defined everywhere (mono.proj sets it for some builds)

Reply:
I tried that and it didn't work for some reason.